Dockerfile

```diff
@@ -0,0 +1,47 @@
+ARG BUILD_IMAGE="nginx-nodejs-supervisord"
+
+# Build next-ui app
+
+FROM ${BUILD_IMAGE} as app_builder
+
+LABEL maintainer="CJSE"
+
+WORKDIR /src/next-ui
+
+COPY ./package*.json ./
+COPY ./scripts/ ./scripts/
+
+RUN npm install
```

Comment:
you can use `npm ci` here https://docs.npmjs.com/cli/v8/commands/npm-ci

Reply:
Resolved in [7aadc3f](https://github.com/ministryofjustice/bichard7-next-ui/pull/5/commits/7aadc3f357b1ee4fe2524a56699df31d3117abed)